pulp_python/app/utils.py

```diff
@@ -1,4 +1,9 @@
 import json
+from django.conf import settings
+from packaging.version import parse
+
+PYPI_LAST_SERIAL = "X-PYPI-LAST-SERIAL"
+PYPI_SERIAL_CONSTANT = 1000000000
```

Comment:
Maybe i am missing the point, but hardcoding a serial number feels wrong.

Reply:
Added a comment for that, thanks.